test/src/e2e_vm_tests/test_programs/should_pass/language/tuple_single_element/src/main.sw

```diff
@@ -0,0 +1,19 @@
+script;
+
+use std::assert::assert;
+
+struct S {
+    t: (u64,)
+}
+
+fn main() -> u64 {
+    let a = S {
+        t: (2,)
+    };
+    let b = match a {
+        S { t } => t,
+    };
+    assert(b.0 == 2);
+
+    1
```

Comment:
Is there a reason to make this change, other than speeding up compile times for this one test? Using assert here is beneficial because it forces the test to include the standard library---including any tuples in the standard library. Assert also adds readability and maintainability---this test can easily be expanded in the future by adding additional asserts and is agnostic to the testing infra used (i.e. to change the test you don't also need to change the test harness)